web/sites/default/settings.php

```diff
@@ -107,6 +120,10 @@
       ],
     ];
   }
+  catch (\Exception $error) {
+    $settings['container_yamls'][] = 'sites/default/redis-unavailable.services.yml';
```

Comment:
I don't see this file in this PR https://github.com/govCMS/govcms8lagoon/blob/develop/.docker/images/govcms8/settings/redis-unavailable.services.yml

Reply:
File added. Fixed